dev-tools/scripts/cloud.sh

```diff
@@ -341,7 +341,7 @@ start(){
     -Dsolr.solrxml.location=zookeeper -Dsolr.log.dir=$CLUSTER_WD_FULL/n${i} $JVM_ARGS")
     FINAL_COMMAND="${SOLR}/bin/solr ${argsArray[@]}"
     echo ${FINAL_COMMAND}
-    ${SOLR}/bin/solr "${argsArray[@]}"
+    ${SOLR}/bin/solr start "${argsArray[@]}"
```

Comment:
@gus-asf I think this is the fix, but I couldn't test the script as I don't have a local ZK setup...   Can you confirm it?